hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -216,12 +214,8 @@ public void run() {
                 }
 
                 iter.remove();
-
-                Exception ex = new TargetDisconnectedException(format(
-                        "Disconnecting from member %s due to heartbeat problems. Current time: %s. Last heartbeat: %s",
-                        connection.getRemoteEndpoint(),
-                        timeToString(System.currentTimeMillis()),
-                        timeToString(connection.getLastHeartbeatMillis())), connection.getCloseCause());
+                Exception ex = newTargetDisconnectedExceptionCausedByHeartBeat(connection.getRemoteEndpoint(),
```

Comment:
The fields/methods are just available on `ClientConnection` which is not available in the member SPI.